cmd/sonobuoy/app/run.go

```diff
@@ -83,7 +85,14 @@ func submitSonobuoyRun(cmd *cobra.Command, args []string) {
 		errlog.LogError(errors.Wrap(err, "could not create sonobuoy client"))
 		os.Exit(1)
 	}
-
+	m := genflags.mode.Get()
+	plugins := []string{}
+	for _, plugin := range m.Selectors {
+		plugins = append(plugins, plugin.Name)
```

Comment:
you could just construct the string with sprintfs here too.

Reply:
you're not wrong, there are many ways to tackle this. I chose this style so I could ignore thinking about when to add the `, ` to the string.